### ISSUE TITLE
layout of views

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#123]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/123) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

we should check if all views on all platforms have the same size on starting mss and add a button to return to this size, or also some selection for differnt once. e.g. 

[640,480]
[...]
[1920,1080]
[free]

This enables the option to get a already processed cache file loaded for a given size and also to use cache files by someone else created.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

While layout sizes easily can be adjusted the size of the image requested on the server is based on some calculations. The values become retrieved from 

in mpl_qtwidget

```
#!python

def getPlotSizePx(self):
        """Determines the size of the current figure in pixels.

        Returns the tuple width, height.
        """
        # (bounds = left, bottom, width, height)
        ax_bounds = self.ax.bbox.bounds
        width = int(round(ax_bounds[2]))
        height = int(round(ax_bounds[3]))
        return width, height


```

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

implementation of views default layout, fixes #123
